services/webnn/ort/error_ort.h

```diff
@@ -24,22 +24,33 @@ namespace webnn::ort {
     }                                                                \
   } while (0);
 
-#define ORT_CALL_FAILED(ort_func)                                  \
-  [&]() -> ort::ScopedOrtStatusPtr {                               \
-    ort::ScopedOrtStatusPtr status(ort_func);                      \
+#define LOG_IF_FAILED(ort_call)                                    \
+  do {                                                             \
+    ort::ScopedOrtStatusPtr status(ort_call);                      \
     if (status) {                                                  \
-      LOG(ERROR) << "[WebNN] Failed to call " << #ort_func << ": " \
+      LOG(ERROR) << "[WebNN] Failed to call " << #ort_call << ": " \
+                 << ort::GetOrtApi()->GetErrorMessage(status);     \
+    }                                                              \
+  } while (0)
+
+#define ORT_CALL_FAILED(ort_call)                                  \
+  ([&]() -> bool {                                                 \
+    ort::ScopedOrtStatusPtr status(ort_call);                      \
+    if (status) {                                                  \
+      LOG(ERROR) << "[WebNN] Failed to call " << #ort_call << ": " \
```

Comment:
Duplication of log code is not ideal. Could you please just use `LOG_IF_FAILED` or `CALL_ORT_FUNC`?
```
ort::ScopedOrtStatusPtr status(CALL_ORT_FUNC(ort_call));
```